### ISSUE TITLE
When using anchor point caching, hidden_states are no longer calculated.

### ISSUE THREAD

**Issue by Lou-Xinyu** [NVlabs/consistory#17]
The subsequent process of calculating the hidden states after this code snippet
            if anchors_cache and anchors_cache.is_cache_mode():
                if self.place_in_unet not in anchors_cache.input_h_cache:
                    anchors_cache.input_h_cache[self.place_in_unet] = {}

                # Hidden states inside the mask, for uncond (index 0) and cond (index 1) prompts
                subjects_hidden_states = torch.stack([x[self.attnstore.last_mask_dropout[width]] for x in hidden_states.chunk(2)])
                anchors_cache.input_h_cache[self.place_in_unet][self.attnstore.curr_iter] = subjects_hidden_states
